.travis.yml

```diff
@@ -2,6 +2,9 @@ language: erlang
 
 otp_release:
   - 18.0
+  - 18.1
+  - 18.2.1
```

Comment:
true.
anyway somehow i managed to have it's on section in the download page.
that shouldn't affect us because it's windows related.